src/extract.js

```diff
@@ -501,19 +501,24 @@ exports.Extractor = class Extractor {
     const reference = new exports.TranslationReference(filename, content, el.startIndex);
     const node = $(el);
 
-    if (this._hasTranslationToken(node)) {
-      if (filename.endsWith('.xml')) {
-        const textArray = [];
-        this.options.attributes.map((keyword) => {
-          if (node.attr(keyword) !== undefined) {
+    if (filename.endsWith('.xml')) {
+      const textArray = [];
+      const elementAttributes = ['label', 'sublabel', 'label-short'];
```

Comment:
are those labels a global standard or specific to your project?

Reply:
specific to atlas project

Reply:
should we make it a project setting then?

Reply:
arguments are defined here: https://github.com/PitcherAG/easygettext/blob/master/src/extract-cli.js#L20

Reply:
maybe even use

Reply:
maybe even use ```// Allow to pass extra attributes, e.g. gettext-extract --attribute v-translate --attribute v-i18n
const extraAttribute = argv.attribute || false;```

Reply:
you mean i should not have this array and pass them as arguments when running the command?